schema/settings.schema.json

```
@@ -60,14 +60,6 @@
             "pointer": true
           }
         },
-        "copyQueryButton": {
```

Will this break things for any users that have that defined? i.e. do we forcefully validate the settings schema, or just ignore errors?

---

No, users will see a validation error the next time they go edit the settings but the error will be ignored and everything should still work OK.

---

Perfect -- thanks for the info!